test/utils/wabt_engine.cpp

```diff
@@ -16,7 +16,7 @@ namespace fizzy::test
 class WabtEngine final : public WasmEngine
 {
     // mutable because getting RefPtr from Ref requires non-const Store
-    mutable wabt::interp::Store m_store{([]() constexpr noexcept {
+    mutable wabt::interp::Store m_store{([]() noexcept {
         wabt::Features features;
         features.disable_multi_value();
```

Comment:
These are not `noexcept`: https://github.com/WebAssembly/wabt/blob/1.0.19/src/feature.h#L39